src/wallet/rpc/backup.cpp

```diff
@@ -431,7 +431,7 @@ RPCHelpMan importpubkey()
 
     std::string strLabel;
     if (!request.params[1].isNull())
-        strLabel = request.params[1].get_str();
+        strLabel = LabelFromValue(request.params[1]);
```

Comment:
What about moving the null check into `LabelFromValue` (if needed with an optional arg to set the default value to something other than `""`)?

Reply:
What do you think of 62535d877c0d9e092499b52c466b982b18b5df39?